.github/workflows/hermes.yaml

```diff
@@ -22,22 +22,26 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-22.04, ubuntu-22.04-arm, macos-14]
+        os: [ubuntu-22.04, ubuntu-22.04-arm, macos-13]
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 60
+    timeout-minutes: 120
     steps:
       - name: Free Disk Space (Ubuntu)
-        if: ${{ matrix.os != 'macos-14' }}
+        if: ${{ matrix.os != 'macos-13' }}
```

Comment:
note that: `macos-14` is arm64 but `macos-13` is amd64.

we need this as the latest madara build steps use docker. `macos-14` (arm64) and `macos-15` (arm64) do not support docker because they don't support nested virtualization.